client/assets/js/admin-controller.js

```diff
@@ -137,5 +145,148 @@ angular.module('portal').controller('adminController', function ($scope, $http,
             // needs implementation
         });
     };
-    
+
+    // get all admin
+    $http.get('/api/user/getUsers').then(function(response) {
```

Comment:
if the email checks are performed in the backend instead of the frontend, then we don't need to fetch all the users, just the admins. so this method probably doesn't need `$scope.comUsers`.